DuckDuckGo/GestureToolbarButton.swift

```diff
@@ -128,14 +128,7 @@ class GestureToolbarButton: UIView {
     }
 
     override func touchesEnded(_ touches: Set<UITouch>, with event: UIEvent?) {
-        guard let touch = touches.first else {
-            imposeReleaseAnimation()
-            return
-        }
-        guard point(inside: touch.location(in: self), with: event) else {
-            imposeReleaseAnimation()
-            return
-        }
```

Comment:
All the above is largely unnecessary - the main scenario it was for is if the user presses down, then moves, then releases, but by that point the long press gesture has already kicked in.